### DISCUSSION TITLE
Global CSS is removed on production build

### DISCUSSION THREAD

**Discussion by WoodNeck** [nuxt/nuxt#10474] [Questions (Nuxt 2)]
Hello, I'm creating the library `@egjs/flicking` and making a guide for the users who use it with nuxt.js.
My module requires a single CSS file to be displayed correctly, so I tried nuxt.js's [Global CSS](https://go.nuxtjs.dev/config-css) to load it.
My config looks like this:
```js
css: [
  "@egjs/flicking/dist/flicking.css"
]
```

It loads on the development environment correctly, but it is being removed when I actually build the static site.
I've tried that on fresh new page built with `npx create-nuxt-app <project-name>`

Any guess on why? Any help is appreciated :)

## How it should look
![image](https://user-images.githubusercontent.com/26213435/170909364-83af99e3-397c-4c79-a5bd-2a5c240a0a7e.png)

## How it looks on production build, due to the missing CSS
![image](https://user-images.githubusercontent.com/26213435/170909399-04101e42-2895-4def-96a1-faa68dd47038.png)

## Styles that I could found on development mode
![image](https://user-images.githubusercontent.com/26213435/170909640-024a568e-dc69-4e5e-a613-deafddae7250.png)
> "@egjs/flicking/dist/flicking.css"

## \<head\> tag of production mode
![image](https://user-images.githubusercontent.com/26213435/170909756-49beec94-5abc-45bd-bc46-803584328082.png)

The repository that reproduce this issue: https://github.com/WoodNeck/nuxt-flicking-global-css
Nuxt version: 2.15.8

**Comment by mahmudtopu3** [comment]
did you get any solution ?

**Reply by WoodNeck** [reply]
I believe it was a webpack issue with `sideEffects`.
Try adding your CSS file in package.json
- https://webpack.js.org/guides/tree-shaking/#mark-the-file-as-side-effect-free
- https://github.com/naver/egjs-flicking/pull/715 (Our fix PR)

```json
{
  "sideEffects": ["*.css"]
}
```

**Reply by mahmudtopu3** [reply]
Strange ! most of the css works fine , I have some basic css like in assets/style.css 
body {
    font-family: 'Lato', sans-serif;
    font-size: 18px;
    font-weight: 400;
    line-height: 2;
    color: #242E39;
}

but in dev environment, everything seems perfect but in production build not!
